### ISSUE TITLE
[Keyboard Navigation : User unable to access 'Insert link' alert dialog which is displayed by invoking 'Insert Link' control present in 'Rich text editor toolbar.

### ISSUE THREAD

**Issue by IyyappanRam2** [ckeditor/ckeditor4#4946]
## Type of report

Keyboard Navigation on CK  Editor Rich Text box

## reproduction steps 

Using Keyboard Controls User unable to access 'Insert link' alert dialog which is displayed by invoking 'Insert Link' control present in 'Rich text editor toolbar.

### Expected result

User should be able to access 'Insert link' alert dialog which is displayed by invoking 'Insert Link' control present in 'Rich text editor toolbar.
Ex: Once the dialog is displayed, keyboard focus should land on the edit field present in the dialog.

### Actual result
User unable to access 'Insert link' alert dialog which is displayed by invoking 'Insert Link' control present in 'Rich text editor toolbar.
Ex: 
1.Once the dialog is displayed, keyboard focus is landing on close(X) button in the Given Feedback dialog.
2.Keyboard focus is not at all moving to the displayed alert dialog

## Other details

* Browser: …
* OS: …
* CKEditor version: …
* Installed CKEditor plugins: …

**Comment by KarolDawidziuk** [comment]
Hi @IyyappanRam2, thank you for the report.

For me, everything looks good and I cannot reproduce your issue but could provide a little more information that could help to confirm this issue like Browser, OS, installed plugins, and the most helpful, working example with steps for reproduction?

I encourage you to use one of our samples to create the reproduction sample, for example:
- [CodePen](https://codepen.io/Comandeer/pen/ExaMgpz?editors=1010)
- [JSBin](https://jsbin.com/keqekef/1/edit?html,js,output)

**Comment by IyyappanRam2** [comment]
> Hi @IyyappanRam2, thank you for the report.
> 
> For me, everything looks good and I cannot reproduce your issue but could provide a little more information that could help to confirm this issue like Browser, OS, installed plugins, and the most helpful, working example with steps for reproduction?
> 
> I encourage you to use one of our samples to create the reproduction sample, for example:
> 
> * [CodePen](https://codepen.io/Comandeer/pen/ExaMgpz?editors=1010)
> * [JSBin](https://jsbin.com/keqekef/1/edit?html,js,output)

Please refer the below codepen sample and in the above description we updated version details , by default keyboard Tab focus not going to CK editor controls.

https://codepen.io/amneerajm/pen/OJjepxj

**Other details**
Browser: MIcrosoft Edge (Chromium) & Google Chrome
OS: Windows 10
CKEditor version: @ckeditor/ckeditor5-react : 3.0.0
Installed CKEditor plugins: @ckeditor/ckeditor5-build-decoupled-document : 25.0.0

**Comment by IyyappanRam2** [comment]
@KarolDawidziuk Do we have any update of this ticket ?

**Comment by KarolDawidziuk** [comment]
@IyyappanRam2 I'm sorry for the late reply.
This report is related to `CKEditor5`, so I'm moving this issue to the appropriate repository.
